### DISCUSSION TITLE
Comprehensive Community Research Plan

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#198] [General]
Hello team!

I have outlined a plan [here](https://docs.google.com/document/d/1EQgj73cbB2Ply1yQmFaMK_Jec6h5MHMu5HANTBvC4Y4/edit?usp=sharing) for the objective of understanding the community more in terms of guiding our development of products both to ensure community feels part of the project and we also keep up with what matter to our users - thanks to @hackfisher for initiating this thought. 

Now the plan is 4 weeks long - week 1 - planning has already been done. So we want to aim for this to work on 3 -4 week basis - and submit the results by the end of Dec 2024.

I would like to have the following teams:

## **1. Research & Survey Team**
**Responsibilities:**
- Design surveys etc.- I have already made the first one for us [here](https://forms.gle/nTd4beUnppVv1b6C8)

**Suggested Roles:**
- Survey Designer: Furqan
- Data Analyst: Furqan - Happy to have a volunteer with me here too!

---

## **2. Community Engagement Team**
**Responsibilities:** (This team gotta hustle around to gather feedback - KPI could be - how many people engaged - filled form - came to talk on stage in calls and spaces.)
- Promote surveys and gather responses across platforms.
- Actively engage with the community, answer queries, and address concerns.  
- Encourage participation using incentives.

**Suggested Roles:**
- Please reach out to me to get this role assigned. 2-3 people in this team is what I am looking for

---

## **3. Strategy & Implementation Team**
**Responsibilities:**
- Consolidate findings from the research team and prioritize insights.  
- Present actionable recommendations to @hackfisher  for product focus.  
- Develop a roadmap for integrating community feedback into product development.  

**Suggested Roles:**
This is going to  be whole team - but I am going to run point here. 

---

## **Reward Team** (MAYBE)
**Responsibilities:**
- Manage and distribute rewards or incentives for participants.  
- Ensure fair allocation and transparency in the reward process.  
- Communicate rules and updates about incentives to participants.  

Now this is a team that might be needed if we decide to boost feedback - but I would like to keep this minimal since incentivizing to give feedback doesn't give the best results. but I am down to redirect some of those quiz rewards

**Comment by Laki389** [comment]
@nblogist 

I am applying for the **2. Community Engagement Team.**

I think the most important thing is the answers from the OG investors who are still with us despite the terrible bear market that was.

Regarding the redirection of dcdao quiz prizes, for your information the budget is between 2000 RING - 3000 RING per week, we are using funds left over from the time when Cindy was still the Community Manager, the funds we spend on quizzes are primarily symbolic and extremely minimal.

On Arbitrum there are 7 000 + 22 000 = 29 000 RING left.

On Polygon there are 1 500 RING.

A total of 30 500 Ring is enough for at best 15 weeks.

**Reply by Laki389** [reply]
> @nblogist
> 
> I am applying for the **2. Community Engagement Team.**
> 
> I think the most important thing is the answers from the OG investors who are still with us despite the terrible bear market that was.
> 
> Regarding the redirection of dcdao quiz prizes, for your information the budget is between 2000 RING - 3000 RING per week, we are using funds left over from the time when Cindy was still the Community Manager, the funds we spend on quizzes are primarily symbolic and extremely minimal.
> 
> On Arbitrum there are 7 000 + 22 000 = 29 000 RING left.
> 
> On Polygon there are 1 500 RING.
> 
> A total of 30 500 Ring is enough for at best 15 weeks.

On second thought, how could rewards be implemented if the survey is anonymous?

**Reply by nblogist** [reply]
Thank you for your question- rewards aren't to be used in start - it is last and a very optional option - we want to take this feedback first from people enthusiastically willing to provide it

**Reply by nblogist** [reply]
SOunds good - Laki - i think your first step should be start reaching out to community - and we can even do small adhoc space on twitter and get the ball rolling 
ping me tomorrow and we can kick it off

CC @2xJDubs (can someone tag partnership dude)

**Reply by nblogist** [reply]
@PD-Ringecosystem

**Reply by nblogist** [reply]
Can we do the space early? weekend or middle next week -  want to work on the data before month ends and have a report ready for @hackfisher

**Reply by 2xJDubs** [reply]
@nblogist I am happy to work with Laki to engage the community with this research.  Unfortunately I will not be available this weekend for a Space so can we do it some time next week?  From there Laki and I will engage the community.  I have requested additional Community Manager rewards funds as we are under budget for November and we could use some of these for engagement.

@Laki389 we could ask participants to show us a screen cap of their submitted response as there will be no connection to their email if this is anon.  Just a thought.

**Reply by 2xJDubs** [reply]
> @nblogist Yes we can do the space earlier.
> 
> Tuesday next week, yeah?
> 
> @2xJDubs @Laki389 

Tuesdays are a no for me.  Only days I can do a space during the week is Wednesday or Friday.

**Comment by nblogist** [comment]
https://github.com/orgs/dcdao/discussions/206

Report published
